src/duties/browserResults.js

```diff
@@ -0,0 +1,57 @@
+let _ = require('lodash/fp')
+
+let hasBrowserErrors = ({ browserResults, fail, message }) => {
+  if (_.isNil(browserResults)) {
+    message('Could not find any browser results')
+    return
+  }
+
+  let { result, summary } = browserResults
+
+  if (result && summary) {
+    let { error, failed, exitCode } = summary
+    if (false === error && 0 === failed && 0 === exitCode) {
+      message('Your PR has no browser errors.  Great job!')
+      return
+    }
+
+    if (failed > 0) {
+      let failedTests = []
```

Comment:
is there a reason why it's starts as an empty array?

Reply:
That's what wanted to know as well. Better solution?

Reply:
From what I understand `_.reduce` requires the `accumulator` as the initial value.  I do agree with you, it definitely could be improved.

Reply:
is it possible to inline it?

Reply:
Just did.

Reply:
Looks way better now.  It would be perfect if we don't have to have that at ;)